module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -167,8 +169,78 @@ Read the full documentation and download cheatsheets (where available) for these
 
 
 
-### `Seurat` and `SCE` object conversion
+## `Seurat` and `SCE` object conversion
+
+The `Seurat` documentation provides a [vignette about converting objects](https://satijalab.org/seurat/articles/conversion_vignette.html) between `SCE` and `Seurat` formats.
+
+In addition, we provide some code examples below for how you can accomplish these conversions.
+For all code examples below, it is assumed that the `SingleCellExperiment` library has been loaded into your R environment:
+
+```r
+library(SingleCellExperiment)
+```
+
+
+### Converting from `Seurat` to `SCE`
+
+The following example code assumes you have a `Seurat` object called `seurat_obj`.
+
+```r
+# Convert Seurat object to SCE object
+sce_object <- Seurat::as.SingleCellExperiment(seurat_obj)
+```
+
+By default, all assays present in the `Seurat` object will be ported into the new `SCE` object.
+To only specify that certain assays are retained, you can optionally provide the argument `assays`, as in: `assays = c("assays", "to", "keep")`.
+
+
+### Converting from `SCE` to `Seurat`
+
+The following example code assumes you are starting with an `SCE` object called `sce_object`.
+
+The function `Seurat::as.Seurat()` can be used to convert an `SCE` object into a `Seurat` object and takes the following arguments:
+
+- The `SCE` object to convert
+- Optional named arguments with the following defaults:
+  - `counts = "counts"` specifies that the `SCE` object contains a `"counts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"counts"` assay in the SCE object, set this argument as `counts = NULL` or rename accordingly, e.g. `counts = "whatever_assay_name_you_are_using"`.
+  - `data = "logcounts"` specifies that the `SCE` object contains a `"logcounts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"logcounts"` assay in the SCE object, set this argument as `data = NULL` or rename accordingly, e.g. `data = "whatever_assay_name_you_are_using"`.
+  - `assay = NULL` specifies that, by default, all assays will be converted.
+    - To specify that an additional assay besides `"counts"` or `"logcounts"` should be converted, include it here as in `assay = "additional_assay_name"`.
```

Comment:
I think I'd flip the order of these two instructions. Because what you are really saying is that if you want to _exclude_ some assays, you need to specify this argument, with all of the assays you want to include (which is confusing). I think it may be less confusing to say something like "The `assays` argument allows you to specify specific addtional assays to include. By default, all assays are converted..." Probably also worth saying if you _don't want additional assays you need to set this to `c()` which is non-intuitive.

Reply:
`assay` apparently does not always mean what we think it means, so I changed this in a different way!